index.d.ts

```diff
@@ -17,8 +17,8 @@ import {
 type Diff<T extends string, U extends string> = ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
 type Omit<T, K extends keyof T> = Pick<T, Diff<keyof T, K>>;
 
-export interface DispatchProp<S> {
-  dispatch?: Dispatch<S>;
+export interface DispatchProp {
+  dispatch?: Dispatch<{}>;
```

Comment:
No need to have a type parameter at all here since we were only instantiating it with `any`. Hard-coding it to `{}` instead of `any` because it's in a contravariant position.